source/mongodb/data-access.txt

```diff
@@ -0,0 +1,120 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in linked MongoDB Atlas clusters from client applications
+using standard MongoDB query language (MQL) syntax with **MongoDB Data Access**
+in the Realm SDKs.
+
+MongoDB Data Access secures data by enforcing your App's role-based access permissions, query filters, and document schemas.
+These features let you construct complex client-side queries with MQL
+while your data remains secure on the server.
+
+When to Use MongoDB Data Access
+-------------------------------
+
+You might want to use MongoDB Data Access in your application if:
+
+- You want to use an API like the MongoDB drivers
+- You want to write database queries in your client code instead of on a server
+- You want to construct ad hoc queries from the client device.
+
+You might want to use MongoDB Data Access **instead of** :ref:`Atlas Device Sync <sync>`
+in an application in these scenarios:
+
+- You are querying documents that are not or cannot be modeled in Realm Database.
+- You want to access collections that are not synced.
+- You do not want to persist data locally.
+- You are using the Realm Web SDK, which does not support Realm Database or Device Sync.
+
+As an alternative to MongoDB Data Access in **client-side** scenarios,
+you can use the following:
+
+- :ref:`Atlas Data API <data-api>`
+- :ref:`Atlas GraphQL API <graphql-api>`
+
+You can also use these APIs without a Realm SDK for your client app's programming
+language as long as the language has a HTTPS or GraphQL client.
+However, these APIs do not feature the same type-safe MQL syntax
+as the Realm SDKs' MongoDB Data Access.
+
+As alternative to MongoDB Data Access in **server-side** scenarios,
+you can use the following:
+
+- Atlas Data API
+- Atlas GraphQL API
+- :driver:`MongoDB driver <>`. If you would like to apply App Services Rules
+  to driver operations, you can use the :ref:`App Services Wire Protocol <wire-protocol>`.
```

Comment:
Per Drew's comment on my other PR, could we emphasize the drivers as our general recommendation for server use cases?

> I would like us to be more specific here.... Something along the lines of "Using the MongoDB Drivers to build on top of Atlas is still the general recommended approach (especially when connecting from a Server) as it will be more performant and flexible."

https://github.com/mongodb/docs-app-services/pull/337#discussion_r1097489375